### ISSUE TITLE
DEB repo: package list refresh fails, error 402 (payment required)

### ISSUE THREAD

**Issue by Miyamoto72** [balena-io/etcher#3511]
- **Etcher version:** 
latest
- **Operating system and architecture:** 
Debian and derivatives
- **Image flashed:** 
not relevant
- **Do you see any meaningful error information in the DevTools?** 
When refreshing the package lists by issueing 'apt update' I get the following error messsages:

> E: Failed to fetch https://dl.cloudsmith.io/public/balena/etcher/deb/linuxmint/dists/ulyssa/InRelease  402  Payment Required [IP: 2600:9000:20e8:5a00:e:f4d2:20c0:93a1 443]
> E: The repository 'https://dl.cloudsmith.io/public/balena/etcher/deb/linuxmint ulyssa InRelease' is no longer signed.
> E: Failed to fetch https://dl.cloudsmith.io/public/balena/etcher/deb/linuxmint/dists/ulyssa/InRelease  402  Payment Required [IP: 2600:9000:20e8:5a00:e:f4d2:20c0:93a1 443]
> E: The repository 'https://dl.cloudsmith.io/public/balena/etcher/deb/linuxmint ulyssa InRelease' is no longer signed.

This makes the repo unusable - maybe cloudsmith has a problem or demands money?

**Comment by eugeneromero** [comment]
Can confirm, same issue on Ubuntu 20.04. Additionally, trying to download the repo install script fails with the same (402) error.

**Comment by lurch** [comment]
Wow, you've chewed through 200GB of bandwidth already?! (I assume Etcher is on the Open-Source tier? https://cloudsmith.com/product/pricing/ )
I guess Etcher is just **too** popular! :rofl:

**Comment by lurch** [comment]
Cloudsmith repo seems to be working again :+1: 
Thanks @thundron

**Comment by thundron** [comment]
We're unblocked for now but we're still working towards a "final" solution as the current one is still fairly limited compared to our ideal usage, I'll leave this open for the time being

**Comment by Frittierapparat** [comment]
It's happening again. A few days ago I installed it just fine, now I get the same "402 Payment Required".

**Comment by oguzkarayemis** [comment]
Same here, apt update returns this error code.

**Comment by jny986** [comment]
Having the issue this morning after working fine for a while. 
Pop Os distro.

`E: Failed to fetch https://dl.cloudsmith.io/public/balena/etcher/deb/pop/dists/hirsute/InRelease  402  Payment Required [IP: 13.226.109.16 443]
E: The repository 'https://dl.cloudsmith.io/public/balena/etcher/deb/pop hirsute InRelease' is no longer signed.`

**Comment by Zorono** [comment]
The same here too.... Ubuntu 21.10 (AMD64)
```
E: Failed to fetch https://dl.cloudsmith.io/public/balena/etcher/deb/ubuntu/dists/impish/InRelease  402  Payment Required [IP: 54.230.114.10 443]
E: The repository 'https://dl.cloudsmith.io/public/balena/etcher/deb/ubuntu impish InRelease' is not signed.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
```

**Comment by mcraa** [comment]
see #3672

**Comment by NunoSempere** [comment]
Not working for me.

**Comment by lurch** [comment]
Yeah, seems to have broken again:
```
Reading package lists... Done                                    
E: Failed to fetch https://dl.cloudsmith.io/public/balena/etcher/deb/ubuntu/dists/bionic/InRelease  402  Payment Required [IP: 99.86.119.33 443]
E: The repository 'https://dl.cloudsmith.io/public/balena/etcher/deb/ubuntu bionic InRelease' is no longer signed.
```

**Comment by ratajs** [comment]
I‘m currently encountering this problem on KDE Neon (Ubuntu 20.04).

**Comment by Josee9988** [comment]
Same here Ubuntu 22.10

**Comment by mcraa** [comment]
It is strange, there is plenty available from the bandwidth for this month.
We'll ping the provider if there was any issue on their side.
Worst case it should reset as the new month starts.
Thanks for the report.

**Comment by ssi-anik** [comment]
Seems like the issue is with the provider. I don't use etcher. *symfony cli* uses cloudsmith and i faced the same issue. 

```
E: Failed to fetch https://dl.cloudsmith.io/public/symfony/stable/deb/ubuntu/dists/focal/InRelease  402  Payment Required [IP: 13.227.138.75 443]
```

**Comment by fermuch** [comment]
Hello. I'm sorry to revive an old thread, but it is happening again. Should I open a new issue?

**Comment by shreefgit** [comment]
Err:11 https://dl.cloudsmith.io/public/balena/etcher/deb/ubuntu kinetic InRelease
  402  Payment Required [IP: 18.67.111.84 443]

**Comment by asterix13** [comment]
I removed any file with reference to cloud cloudsmith  and balena etcher in the folder "/etc/apt/sources.list.d"

worked for me

**Comment by LarnuUK** [comment]
> I removed any file with reference to cloud cloudsmith and balena etcher in the folder "/etc/apt/sources.list.d"
> 
> worked for me

That's not really a "solution", as now once the repository is fixed you won't obtain any updates as you've removed the repo from `apt`. You would have been "better" off disabling the repo for now in my opinion, which can by done by prefixing the repo address in the file with a hash (`#`) or using your package manager. Then once it's confirmed in this thread, or a different one, you can re-enable the repo. You'll instead need to re-add the repository(s) now once the issue is resolved.

**Comment by asterix13** [comment]
Oh right, should've thought about that. However I did not need to balena etcher anyways so I removed it out of sheer frustration.

**Comment by drbornot** [comment]
I can confirm the issue since yesterday I couldn't fetch the repo for updates

```
E: Failed to fetch https://dl.cloudsmith.io/public/balena/etcher/deb/pop/dists/jammy/InRelease  402  Payment Required [IP: 143.204.89.12 443]
E: The repository 'https://dl.cloudsmith.io/public/balena/etcher/deb/pop jammy InRelease' is no longer signed.
```

**Comment by Argentino84** [comment]
Same issue on Ubuntu focal.

E: Failed to fetch https://dl.cloudsmith.io/public/balena/etcher/deb/ubuntu/dists/focal/InRelease 402  Payment Required [IP: 2600:9000:2190:be00:e:f4d2:20c0:93a1 443]
E: The repository 'https://dl.cloudsmith.io/public/balena/etcher/deb/ubuntu focal InRelease' is no longer signed.

**Comment by lenhatquang2512** [comment]
same issue on Ubuntu 20.04 LTS.

```
E: Failed to fetch https://dl.cloudsmith.io/public/balena/etcher/deb/ubuntu/dists/focal/InRelease  402  Payment Required [IP: 18.160.249.30 443]
E: The repository 'https://dl.cloudsmith.io/public/balena/etcher/deb/ubuntu focal InRelease' is no longer signed.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
```

**Comment by tsantor** [comment]
Same issue in Ubuntu 22.04 LTS.

```
E: Failed to fetch https://dl.cloudsmith.io/public/balena/etcher/deb/ubuntu/dists/jammy/InRelease  402  Payment Required [IP: 18.165.32.112 443]
E: The repository 'https://dl.cloudsmith.io/public/balena/etcher/deb/ubuntu jammy InRelease' is no longer signed.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
W: https://packagecloud.io/slacktechnologies/slack/debian/dists/jessie/InRelease: Key is stored in legacy trusted.gpg keyring (/etc/apt/trusted.gpg), see the DEPRECATION section in apt-key(8) for details.
```

**Comment by xarpy** [comment]
Same issue on Ubuntu 22.04 LTS:

```
E: Falhou ao buscar https://dl.cloudsmith.io/public/balena/etcher/deb/ubuntu/dists/jammy/InRelease  402  Payment Required [IP: 108.139.134.25 443]
E: O repositório 'https://dl.cloudsmith.io/public/balena/etcher/deb/ubuntu jammy InRelease' já não está assinado.
N: Atualizações a partir de tal repositório não podem ser feitas de forma segura e estão, portanto, desativadas por definição.
N: See apt-secure(8) manpage for repository creation and user configuration details.
W: https://packagecloud.io/slacktechnologies/slack/debian/dists/jessie/InRelease: Key is stored in legacy trusted.gpg keyring (/etc/apt/trusted.gpg), see the DEPRECATION section in apt-key(8) for details.

```

**Comment by GeanVas** [comment]
It's on Debian Bullseye too
```
E: Failed to fetch https://dl.cloudsmith.io/public/balena/etcher/deb/debian/dists/bullseye/InRelease  402  Payment Required [IP: 65.8.248.50 443]
E: The repository 'https://dl.cloudsmith.io/public/balena/etcher/deb/debian bullseye InRelease' is no longer signed.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.

```

**Comment by lurch** [comment]
See the most recent comments in #4034

**Comment by aethernet** [comment]
Hello,

We're moving out of Cloudsmith as continuing to host the package there would be too costly for us.

We're very thankful for the service and help CloudSmith provided, and would recommend their product for smaller open source app and private packages anytime!

At the moment we don't have the bandwidth (pun intended) to set up an alternative repository but might be moving to self hosted (probably CloudFlare R2 + workers) in the upcoming weeks.

So sorry for the inconvenience and thanks you for your patience.

Note that you can always get a .deb and .rpm package in the release section of the github repository.

**Comment by pmo-19** [comment]
Unfortunately no I can't since in My Ubuntu Focal 20.04 I have package balena-etcher-electron installed which is not available in your GitHub release section ...

**Comment by RaibeartRuadh** [comment]
I confirm that after removal from source list balena-etcher, the update proceeded successfully. The "402 Payment Required" message no appeared.
Ubuntu 22 -> 23

**Comment by omar55221** [comment]
a solution for Ubuntu 18.04 was to remove the balena etcher by following step no. 5 
https://www.cyberithub.com/how-to-install-and-use-balenaetcher-on-debian-11-bullseye/#Step_5_Uninstall_balenaEtcher
and added sudo before rm commands

**Comment by gamesover** [comment]
@omar55221 it works for me. Thank you
